index.html

```diff
@@ -161,6 +161,9 @@ <h2 class="ui dividing header">Links</h2>
           <li class="es2"><a href="https://www.elastic.co/guide/en/elasticsearch/guide/2.x/index.html">Elasticsearch: The Definitive Guide</a>, official book;</li>
           <li class="es5 es6"><a href="https://www.elastic.co/guide/en/elasticsearch/guide/master/index.html">Elasticsearch: The Definitive Guide</a>, official book (not up to date);</li>
 
+          <li class="es1 es2 es3 es4 es5 es6"><a href="https://www.elastic.co/downloads/past-releases/#elasticsearch">Download Elasticsearch</a>, official download page;</li>
```

Comment:
es3 and es4 do not exists :grimacing: (there is no such version)

Reply:
Sorry for that

Reply:
@damienalexandre Review again